### ISSUE TITLE
Block of flats with building=apartments not rendering.

### ISSUE THREAD

**Issue by kpv8** [jetelain/ArmaRealMap#155]
I have noticed that all long block of flats buildings with building=apartments tag on osm are not showing on generated terrain. 

Example location: https://www.openstreetmap.org/#map=19/51.54500/23.54962

**Comment by jetelain** [comment]
Hi,
Those buildings are detected, but default asset configuration have no building with appropriate dimensions, so it's dismissed.
You will have to create you own asset configuration, and to add buildings from a mod or to create a composition with matching size.

**Comment by Hauzmanus** [comment]
I recommend taking a look at RHSTERRACORE, it has apartment buildings in it, which are highly detailed.

**Comment by Autoclock** [comment]
agreed with the above, but keep in mind some assetsmight be in separate pieces and thus would need to be properly lined up in VR EDEN and imported

**Comment by Hauzmanus** [comment]
Absolutely keep in mind some are seperate pieces. RHSTERRACORE mod has an apartment building though that is full building, no additional building placement is required. You can use that in automated generation. Obviously you will need to remove these buildings from small towns and etc until the tagging system is improved to support buildings as well. :)
